Makefile.work

```diff
@@ -38,6 +38,13 @@
 #  * SONIC_DPKG_CACHE_METHOD: Specifying method of obtaining the Debian packages from cache: none or cache
 #  * SONIC_DPKG_CACHE_SOURCE: Debian package cache location when cache enabled for debian packages
 #  * BUILD_LOG_TIMESTAMP: Set timestamp in the build log (simple/none)
+#  * SONIC_VERSION_CONTROL_COMPONENTS : In the process of SONiC build, there are some source code,
+#  *                               which are downloaded by third utils. ex: git,wget,pip,docker...
+#  *                    By default build will use version file in 'files/build/versions/' to control third utils version.
+#  *                    if we specify one component, build will use latest version and record it in a new file
+#  *                    ex: git's version file is files/build/versions/versions-git, if ${var} is "pip,git"
+#  *                        git clone will use the latest commit, and copy the commit ID to files/build/versions/versions-git-latest.
+#  *                        When build succeeded, you can replace versions-git by versions-git-latest, and commit the changes.
 #
```

Comment:
What about the `Default` and `Values` fields?

Reply:
changed